app/views/groups/show.html.slim

```diff
@@ -32,13 +35,10 @@ div
 
 .p-4.my-4.shadow-md
   p.mb-4 掲示板
-  .overflow-y-auto.max-h-80
-    - if @posts.any?
-      - @posts.each do |post|
-        == render partial: 'posts/post', locals: { post:, group: @group }
-    - else
-      p
-        | まだ投稿はありません。
+  #posts.overflow-y-auto.max-h-80
+    == render partial: 'posts/post', collection: @posts, locals: { group: @group }
+    .hidden.only:block
+      | まだ投稿はありません。
```

Comment:
- [Empty states with CSS and the :only-child selector | Sebastian De Deyne](https://sebastiandedeyne.com/empty-states-with-css-and-the-only-child-selector)
- [Display content with the 'only' class in Tailwind CSS - DEV Community](https://dev.to/thomasvanholder/display-content-with-the-only-class-in-tailwindcss-4glh)
- [Hover, focus, and other states - Core concepts - Tailwind CSS](https://tailwindcss.com/docs/hover-focus-and-other-states#only-child)